keyboards/ergoslab/info.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Your key sequence is out of order here:

![ergoslab](https://user-images.githubusercontent.com/18669334/71549812-a63c3780-2978-11ea-88b9-eb163f18b92c.PNG)

Try this:

```json
{
  "keyboard_name": "ergoslab",
  "url": "https://github.com/tomsaleeba/ergoslab",
  "maintainer": "tomsaleeba",
  "width": 13,
  "height": 5.25,
  "layouts": {
    "LAYOUT_ergoslab": {
      "key_count": 52,
      "layout": [
        {"label":"Q", "x":0, "y":0.625},
        {"label":"W", "x":1, "y":0.125},
        {"label":"E", "x":2, "y":0},
        {"label":"R", "x":3, "y":0.125},
        {"label":"T", "x":4, "y":0.25},
        {"label":"Y", "x":8, "y":0.25},
        {"label":"U", "x":9, "y":0.125},
        {"label":"I", "x":10, "y":0},
        {"label":"O", "x":11, "y":0.125},
        {"label":"P", "x":12, "y":0.625},
        {"label":"A", "x":0, "y":1.625},
        {"label":"S", "x":1, "y":1.125},
        {"label":"D", "x":2, "y":1},
        {"label":"F", "x":3, "y":1.125},
        {"label":"G", "x":4, "y":1.25},
        {"label":"H", "x":8, "y":1.25},
        {"label":"J", "x":9, "y":1.125},
        {"label":"K", "x":10, "y":1},
        {"label":"L", "x":11, "y":1.125},
        {"label":":", "x":12, "y":1.625},
        {"label":"Z", "x":0, "y":2.625},
        {"label":"X", "x":1, "y":2.125},
        {"label":"C", "x":2, "y":2},
        {"label":"V", "x":3, "y":2.125},
        {"label":"B", "x":4, "y":2.25},
        {"label":"N", "x":8, "y":2.25},
        {"label":"M", "x":9, "y":2.125},
        {"label":"<", "x":10, "y":2},
        {"label":">", "x":11, "y":2.125},
        {"label":"?", "x":12, "y":2.625},
        {"label":"Leader", "x":0, "y":3.625},
        {"label":"Alt", "x":1, "y":3.125},
        {"label":"Del", "x":2, "y":3},
        {"label":"GUI", "x":3, "y":3.125},
        {"label":"Shift", "x":4, "y":3.25},
        {"label":"Ctrl", "x":5, "y":3.25},
        {"label":"Numb", "x":7, "y":3.25},
        {"label":"Space", "x":8, "y":3.25},
        {"label":"", "x":9, "y":3.125},
        {"label":"", "x":10, "y":3},
        {"label":"\u2191", "x":11, "y":3.125},
        {"label":"\u2192", "x":12, "y":3.625},
        {"x":1, "y":4.25},
        {"label":"Swap", "x":2, "y":4.25},
        {"label":"Esc", "x":3, "y":4.25},
        {"label":"Media", "x":4, "y":4.25},
        {"label":"Brkt", "x":5, "y":4.25},
        {"label":"Tab", "x":7, "y":4.25},
        {"label":"AltTab", "x":8, "y":4.25},
        {"label":"Bspc", "x":9, "y":4.25},
        {"label":"Swap", "x":10, "y":4.25},
        {"x":11, "y":4.25}
      ]
    }
  }
}
```